avaframe/runScripts/runTestFP.py

```diff
@@ -10,7 +10,6 @@
 import avaframe.in3Utils.fileHandlerUtils as fU
 from avaframe.in1Data import getInput as gI
 import avaframe.com1DFA.com1DFA as com1DFA
-from avaframe import runCom1DFA
 import avaframe.ana1Tests.FPtest as FPtest
```

Comment:
This script fails with:
`ModuleNotFoundError: No module named 'avaframe.out3Plot.makePalette'`

and the com1DFA import is also not the correct one, right?

Reply:
I remove the import as it is not required and changed the import of com1DFA, also removed the hSPH as it is not added to the particles anymore - but still fails -made an issue out of it